src/platforms/common/performance/instrumentation/performance-metrics.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Performance Metrics
+sidebar_order: 20
+supported:
+  - javascript
+notSupported:
+  - javascript.cordova
+  - javascript.electron
+  - react-native
+  - dotnet
+  - python
+  - go
+  - java
+  - android
+  - ruby
+  - java.spring-boot
+  - apple
+  - unity
+  - dart
+  - flutter
+  - rust
+  - native
+  - php
+  - native.breakpad
+  - native.crashpad
+  - native.minidumps
+  - unreal
+  - native.wasm
+description: "Learn how to attach performance metrics to your transactions."
+---
+
+The Sentry SDK supports sending performance metrics data to Sentry. These are numeric values attached to transactions that are aggregated and displayed in Sentry.
+
+<PlatformContent includePath="performance/automatic-performance-metrics" />
+
+In addition to automatic performance metrics, the SDK supports setting custom performance metrics on transactions. This allows you to define metrics that are important to your application and send them to Sentry.
+
+To set a performance metric, supply a name (string), value (numeric - float, integer, etc.), and unit (string). Sentry supports adding arbitrary custom units, but we recommend using one of the [supported units listed below](./#supported-performance-metric-units).
+
+<PlatformContent includePath="performance/custom-performance-metrics" />
+
+## Supported Performance Metric Units
+
+Units augment metric values by giving them a magnitude and semantics. Sentry leverages units associated with performance metrics to apply built-in unit conversions and controls in the product. For values that are unitless, you can supply an empty string or `none`.
```

Comment:
```suggestion
Units augment metric values by giving meaning to what otherwise might be abstract numbers. Adding units also allows Sentry to offer controls - unit conversions, filters, etc. - based on those units. For values that are unitless, you can supply an empty string or `none`.
```

I am checking whether or not the "filters" bit there is true, and will update if necessary.

Reply:
For some reason it's not letting me commit this suggestion. Going to resolve the convo and try again.